src/wmnds/components/content-tiles/related-content/_related-content.scss

```diff
@@ -0,0 +1,12 @@
+.wmnds-content-tile {
+  &__list {
+    padding: 0;
+    list-style: none;
+
+    &-links {
```

Comment:
```suggestion
    &__links {
```

Reply:
This is the last change I believe.

Reply:
wmnds-content-tile__list-links so to apply changes to links, I'd have to put &-links. There is no class &__links. So should I leave it as it is?

Reply:
In that case you should make the scss class....

&__list-links {
}